packages/app/src/components/core/Navbar.react.js

```diff
@@ -26,6 +26,7 @@ export default class Navbar extends React.Component {
           <Link to="/subscribers">Subscribers</Link>
           <Link to="/periods">Periods</Link>
           <Link to="/anj-balances">ANJ</Link>
+          { !admin.id && <Link to="/admin">Login</Link> }
```

Comment:
Let's remove this line, it is not necessary to show people this is available

Reply:
ok removed